wom/src/main/scala/wom/util/YamlUtils.scala

```diff
@@ -38,7 +38,7 @@ object YamlUtils {
             maxDepth: Int Refined NonNegative = defaultMaxDepth
            ): Either[ParsingFailure, Json] = {
     try {
-      val yamlConstructor = new SafeConstructor()
+      val yamlConstructor = new SafeConstructor(loaderOptions)
```

Comment:
No-arg constructor was deprecated in 1.31 or 1.32. Now using suggested replacement.